product_search/product_search.py

```diff
@@ -0,0 +1,46 @@
+"""Program that returns inventory details based for an inventory item"""
+
+import sys
+from data_management import access_json_data
+
+data = access_json_data(sys.argv[1])
```

Comment:
If I remove this data variable here, then the program will not run properly. Should I make this a constant?